### ISSUE TITLE
OPERS top nav spacing issues

### ISSUE THREAD

**Issue by teresasilva71** [ucsc/wcms-design-templates#426]
INC0547074
They already put a ticket in:
![opers spacing](https://user-images.githubusercontent.com/7795841/37936953-6fa83680-310c-11e8-94e0-5ca099fc338a.jpg)

**Comment by teresasilva71** [comment]
AND the yellow arrow icon isn't appearing.

**Comment by knice** [comment]
They have `width` attributes in the HTML of their nav items. They need to remove the widths and it will work fine.

**Comment by knice** [comment]
Icons only appear when the dropdown text is wrapped in a `<span>`. There is no way around this one. It's been this way all along. That is why some dropdown menus show the icon and some don't.

**Comment by teresasilva71** [comment]
Makes sense but clients aren't actually coding that into their top nav so I'm not sure how to make this consistent. 
What is the span class required for this? Was it default and then clients deleted it over time?

**Comment by knice** [comment]
When we first built main navs for people, they could only have dropdowns in the last nav item. We'd use a `<span>` in place of an `<a>` for the last nav item, and then put the `<ul>` of dropdown items immediately after it.

I might be able to write some Javascript to hunt down a tags with uls right after them. But that is not a priority while close bugs.

**Comment by teresasilva71** [comment]
See INC0553360. We have a client asking about the drop down arrows.
I'm happy to fix these as they come in....just need to know if it's simple HTML I can add to a top nav block?
